.gitignore

```diff
@@ -24,6 +24,7 @@ target/
 # Node files
 /accounts
 /data
+*.mac
```

Comment:
nit: I'd add a statement re keyfiles

Reply:
`Node files` is a bit ambiguous, and additional detail would be helpful. Feel free to ignore, micro nit